### ISSUE TITLE
Gain normalized or not

### ISSUE THREAD

**Issue by r9y9** [r9y9/MelGeneralizedCepstrums.jl#6]
Maybe adding a type parameter to MelGeneralizedCepsturm?

``` julia
abstract GainForm
type NormliazedGain <: GainForm
end
type UnNormalizedGain <: GainForm
end
...

immutable MelGeneralizedCepstrum{F<:Frequency,L<:Log,G<:GainForm, T<:FloatingPoint,N} <: AbstractMelGeneralizedCepstrumArray{F,L,T,N}
...
```

**Comment by r9y9** [comment]
will be fixed by #8

**Comment by r9y9** [comment]
`SpectralParamState` manages me-generalized cepstrum values are gain-normalized or not.
